lib/components/src/card/card-base.ts

```diff
@@ -17,29 +14,46 @@ export class CardBase extends LitElement {
    */
   @property()
   subTitle = '';
-
+  /**
+   * Displays the action bar
+   */
+  @property({type: Boolean}) actionBar = false;
   /**
    * Style the card as an outline variant
    */
   @property({type: Boolean}) outlined = false;
-   
   override render() {
     const classes = {
       'mdc-card': true,
       'mdc-card--outlined': this.outlined,
     };
     return html`
+      <style>
+        hr {
+          opacity: 0.12;
+          width: 100%;
+          margin: 0;
+        }
+        .mdc-typography--headline5 {
+          color: var(--mdc-theme-text-primary-on-background);
+        }
+      </style>
       <div class="${classMap(classes)}">
+      ${this.cardTitle ? 
+        html`
+          <div class="mdc-card__actions mdc-typography--headline5">${this.cardTitle}</div>
+          <hr>
```

Comment:
This should be css on the card title, not an element in the markup

Reply:
Sorry I'm not sure what you mean, can you explain a little more? Are you talking about the `<hr>`?

Reply:
Yep, the divider should be applied via css to the bottom of whatever element it's separating, not as a separate element in the html.